_appmap/recording.py

```diff
@@ -80,6 +80,22 @@ def __exit__(self, exc_type, exc_value, tb):
             self.exit_hook(self)
         return False
 
+def check_call_return_stack_order(events):
```

Comment:
As with the django PR, this doesn't belong here. Checking it in the test case is ok, though, so moving it to test_events.py is fine.